packages/theme-dark-extension/style/urls.css

```diff
@@ -82,6 +82,14 @@
   --jp-icon-upload: url('./lab/api/themes/jupyterlab-theme-dark-extension/icons/md/upload.svg');
   --jp-icon-question-mark: url('./lab/api/themes/jupyterlab-theme-dark-extension/icons/jupyter/questionmark.svg');
   --jp-icon-chat: url('./lab/api/themes/jupyterlab-theme-dark-extension/icons/md/chat.svg');
+  --jp-icon-checkmark: url('./lab/api/themes/jupyterlab-theme-dark-extension/icons/md/checkmark.svg');
```

Comment:
We need corresponding entries in the embed files: https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-dark-extension/style/embed.css